src/com/futurewei/alcor/controller/db/redis/RedisCache.java

```diff
@@ -0,0 +1,47 @@
+package com.futurewei.alcor.controller.db.redis;
+
+import com.futurewei.alcor.controller.db.ICache;
+import org.springframework.data.redis.core.HashOperations;
+import org.springframework.data.redis.core.RedisTemplate;
+
+import java.util.Map;
+
+public class RedisCache<K, V> implements ICache<K, V> {
+    private HashOperations hashOperations;
+    private String key;
```

Comment:
Shall we use other name like "cache_name" to be differentiated with the KEY in K-V?